xml/System/Int16.xml

```diff
@@ -2274,10 +2274,10 @@
         <Parameter Name="result" Type="System.Int16" RefType="out" Index="1" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="s">To be added.</param>
-        <param name="result">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="s">A span containing the characters representing the number to convert. The span is interpreted using the style specified by <paramref name="style" />.</param>
```

Comment:
This one doesn't have a param named style

Reply:
Nice catchg @Thraka , thank you! Normally I would expect the reviewer to block merging for such a suggestion. Would you mind blocking next time if you find an issue like this? That would help ensure we don't accidentally merge incorrect documentation. :)

Reply:
By me not blocking I was trying to let you fix those without me having to come back and reapprove. 😄 Since these are minor and you can fix them and then you're done with this completely, I generally note and unblock. But I'll be more frugal 😄